sqlite_tests/Dockerfile

```diff
@@ -0,0 +1,46 @@
+FROM golang:1.19 as builder
+
+WORKDIR /src/litefs
+COPY ../. .
+
+ARG LITEFS_VERSION=
+ARG LITEFS_COMMIT=
+
+RUN --mount=type=cache,target=/root/.cache/go-build \
+	--mount=type=cache,target=/go/pkg \
+	go build -ldflags "-s -w -X 'main.Version=${LITEFS_VERSION}' -X 'main.Commit=${LITEFS_COMMIT}' -extldflags '-static'" -tags osusergo,netgo,sqlite_omit_load_extension -o /usr/local/bin/litefs ./cmd/litefs
+
+FROM debian:buster-20221219-slim
+RUN set -ex \
+  && apt-get update \
+  && apt-get upgrade -y --no-install-recommends \
+  && apt-get install -y curl ca-certificates unzip \
+  && apt-get install -y build-essential tcl tcl-dev zlib1g-dev \
+  && apt-get install -y kmod procps nano \
+  && apt-get install -y fuse 
+
+RUN adduser --disabled-password --gecos "" build
```

Comment:
sqlite tests won't run as `root`, so need to create another user to use to run the tests